src/assets/translations/fr/main.json

```diff
@@ -557,14 +598,21 @@
     },
     "deposit": {
       "amountToDeposit": "Montant à déposer",
+      "depositInto": "Déposer dans %{opportunité}",
```

Comment:
Interpolation variables shouldn't be translated

Reply:
Addressed in https://github.com/shapeshift/web/pull/2220/commits/b21538aa8648db3efa253c57ceacc30df3f9871b